config/crd/bases/modelregistry.opendatahub.io_modelregistries.yaml

```diff
@@ -45,30 +45,38 @@ spec:
         description: ModelRegistry is the Schema for the modelregistries API
         properties:
           apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation
-              of an object. Servers should convert recognized schemas to the latest
-              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            description: |-
+              APIVersion defines the versioned schema of this representation of an object.
+              Servers should convert recognized schemas to the latest internal value, and
+              may reject unrecognized values.
+              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
             type: string
           kind:
-            description: 'Kind is a string value representing the REST resource this
-              object represents. Servers may infer this from the endpoint the client
-              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            description: |-
+              Kind is a string value representing the REST resource this object represents.
+              Servers may infer this from the endpoint the client submits requests to.
+              Cannot be updated.
+              In CamelCase.
+              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
             type: string
           metadata:
             type: object
           spec:
-            description: ModelRegistrySpec defines the desired state of ModelRegistry.
+            description: |-
+              ModelRegistrySpec defines the desired state of ModelRegistry.
               One of `postgres` or `mysql` database configurations MUST be provided!
```

Comment:
We've verified that MariaDB is supported as well, should we update this?

Reply:
afair, MariaDB is not recommended for production by the DSP team, so I'm inclined to leave this as is.